app/config/site-config.yml

```
@@ -6,13 +6,35 @@
   transmission_method: shared_email
   transmission_method_configuration:
     email: <%= ENV['NYC_HRA_EMAIL'] %>
+  sso:
+    client_id:     <%= ENV["AZURE_NYC_DSS_CLIENT_ID"] %>
+    client_secret: <%= ENV["AZURE_NYC_DSS_CLIENT_SECRET"] %>
+    tenant_id:     <%= ENV["AZURE_NYC_DSS_TENANT_ID"] %>
+    redirect_uri:  <%= ENV["AZURE_NYC_DSS_REDIRECT_URI"] %>
+    response_type: "id_token"
+    response_mode: "form_post"
+    scope: "openid"
+    state: "12345"
+    nonce: "678910"
+    name: "nyc_dss"
```

Where did we get this list of config options from? I'm cross-referencing with [the Azure AD gem](https://github.com/RIPAGlobal/omniauth-azure-activedirectory-v2/tree/master?tab=readme-ov-file#configuration-options) and there's a bunch here that aren't listed. I just tried this locally with the following config and it seemed to work:

```suggestion
    client_id:     <%= ENV["AZURE_NYC_DSS_CLIENT_ID"] %>
    client_secret: <%= ENV["AZURE_NYC_DSS_CLIENT_SECRET"] %>
    tenant_id:     <%= ENV["AZURE_NYC_DSS_TENANT_ID"] %>
    scope: "openid"
    name: "nyc_dss"
```

I was kind of surprised to not need `redirect_uri`, nor the response_type ones, but I think they get added automatically. Can you try that and see if it works? I looked at this because it includes `state` and `nonce` which definitely feels wrong. (I believe those are meant to be parameters passed in uniquely for each authentication attempt.)